### DISCUSSION TITLE
Volume slider or selector within app

### DISCUSSION THREAD

**Discussion by knakamura8** [libre-tube/LibreTube#5822] [Ideas]
### Describe your suggested feature

**Suggestion**

Volume slider or selector within the app, such as under the `Playback speed` area.  Would be useful when in both video mode as well as in audio/background mode.

**Motivation**

Having music on or watching a video in PiP while on a non-Phone call or when using another app (such as a game with some sound effects) and wanting to adjust the volume of the content playing in the video or in the background (audio ducking of course being disabled first).  Some other media players on Android allow for this within the apps themselves.  I am also aware that some Android versions/builds allow for per-app volume sliders, but many do not.  I am unsure as to how the audio processor is currently set up or what APIs are being used, but this feature I think would be very useful.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
We already have swipe gestures for that, I don't think additional controls make sense.

**Comment by knakamura8** [comment]
The swipe gestures are for global volume, though, I think?  I could be incorrect, here.
